sdk/python/dist/pulumi-analyzer-policy-python

```diff
@@ -1,2 +1,43 @@
 #!/bin/sh
-python3 -u -m pulumi.policy $@
\ No newline at end of file
+
+# Parse the -virtualenv command line argument.
+virtualenv=""
+for arg in "$@"
+do
+    case $arg in
+        -virtualenv=*)
+        virtualenv="${arg#*=}"
+        break
+        ;;
+    esac
+done
+
+if [ -n "${virtualenv:-}" ] ; then
+    # Remove trailing slash.
+    virtualenv=${virtualenv%/}
+
+    # Make the path absolute (if not already).
+    case $virtualenv in
+        /*) : ;;
+        *) virtualenv=$PWD/$virtualenv;;
+    esac
+
+    # If python exists in the virtual environment, set PATH and run it.
+    if [ -f "$virtualenv/bin/python" ]; then
+        # Update PATH and unset PYTHONHOME.
+        PATH="$virtualenv/bin:$PATH"
+        export PATH
+        if [ -n "${PYTHONHOME:-}" ] ; then
+            unset PYTHONHOME
+        fi
+
+        # Run python from the virtual environment.
+        "$virtualenv/bin/python" -u -m pulumi.policy "$1" "$2"
+    else
+        echo "\"$virtualenv\" doesn't appear to be a virtual environment"
+        exit 1
+    fi
+else
+    # Otherwise, just run python3.
+    python3 -u -m pulumi.policy "$1" "$2"
```

Comment:
We look for `-virtualenv` and use it inside these scripts, but don't pass it along (and any other options; yet) to the `pulumi.policy` module because, unfortunately, we currently check for exactly 2 arguments there, instead of a minimum of 2. So we continue to pass the two arguments we passed previously, otherwise older versions of `pulumi-policy` package will fail when it receives more than 2 args.

I've updated the `pulumi.policy` module to look for a minimum of 2 args, so in the future, we could consider passing all the args along (when we don't care as much about older versions of `pulumi` package. (I've also done the same for Node.js, which is currently looking for an exact number of args, rather than a minimum).